internal/cmd/run/run.go

```diff
@@ -26,6 +26,9 @@ const (
 	DisableTraceFlagErrorMessage    = "impossible to get flag --disable-trace: %s"
 	DryRunFlagErrorMessage          = "impossible to get flag --dry-run: %s"
 	UnableParseDurationErrorMessage = "unable to parse duration: %s"
+
+	//
+	//defaultTargetProcessingDelay = 100 * time.Millisecond
```

Comment:
In golang, default values depends on the type, so for the time.Duration is "0s". These lines are not needed

```suggestion
```